pages/views/_layouts/html5.njk

```diff
@@ -8,6 +8,7 @@
 </head>
 <body>
 {% block body %}{% endblock %}
+{% block extra_body %}{% endblock %}
```

Comment:
maybe add this block to basic.njk, line 42? it will be more transparent logic of extensions and the code will be before adding scripts

Reply:
OK, thanks!